Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -338,7 +343,7 @@ DataProcessorSpec getCATrackerSpec(bool processMC, bool caClusterer, std::vector
         validInputs.set(sector);
         datarefs[sector] = ref;
         if (caClusterer) {
-          inputDigits[sector] = pc.inputs().get<const std::vector<o2::tpc::Digit>>(inputLabel);
+          inputDigits[sector] = std::move(pc.inputs().get<const std::vector<o2::tpc::Digit>>(inputLabel));
```

Comment:
Even with ``move`` an extra copy is created. I guess using the ``gsl::span`` right now will not work since the digits are provided by the RootTreeReader, which enforces root serialization even for PODs? @matthiasrichter is there in news on fixing this?